### PR TITLE
[W  8][CS2103AUG2017-F11-B2]Jiasheng

### PR DESCRIPTION
**Pull request by AngularJiaSheng**
Full implementation and documentation for Tag Delete. However, System test still not done.

### CODE REVIEW

**Comment by hansiang93** [comment]
Gonna have to wait till all tests are written before merging

File: src/main/java/seedu/address/logic/commands/DeleteTagCommand.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Deletes a tag from all contacts in the address book.
+ */
+public class DeleteTagCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "tagDelete";
+    public static final String COMMAND_ALIAS = "t";
```
Comment (zhoukai07) [inline src/main/java/seedu/address/logic/commands/DeleteTagCommand.java]: I don't think "t" is a clear label for tagDelete, perhaps using "td" would be better?

File: src/main/java/seedu/address/logic/parser/DeleteTagCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class DeleteTagCommandParser implements Parser<DeleteTagCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteTagCommand
+     * and returns an DeleteTagCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteTagCommand parse(String args) throws ParseException {
+        try {
+            int lengthOfUserInput = args.split(" ").length;
+            if (lengthOfUserInput > 2 || lengthOfUserInput <= 1) {
```
Comment (zhoukai07) [inline src/main/java/seedu/address/logic/parser/DeleteTagCommandParser.java]: is it the same as "if (lengthOfUserInput != 2)" ?
did you consider trimming the args first?

**Review by zhoukai07 — CHANGES_REQUESTED** [review]
hey I think these are some minor changes that could be done.

**Comment by AngularJiaSheng** [comment]
oh man i think i just screwed everything up OMGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG

**Pull request closed without merging**